zombies.js

```diff
@@ -79,6 +90,43 @@
  * @property {method} getMaxHealth         Returns private variable `maxHealth`.
  */
 
+// class Player {
+//   constructor(name, health, strength, speed) {
+//     this.name = name;
+//     this.health = health;
+//     this.strength = strength;
+//     this.speed = speed;
+//     this.isAlive = true;
+//     this.equipped = false;
+//     this._pack = [];
+//     this._maxHealth = health;
+//   }
+
+//   getPack() {
+//     return this._pack;
+//   }
+
+//   getMaxHealth() {
+//     return this._maxHealth;
+//   }
+
+function Player(name, health, strength, speed) {
+  this.name = name;
+  this.health = health;
+  this.strength = strength;
+  this.speed = speed;
+  this.isAlive = true;
+  this.equipped = false;
+  this._pack = [];
+  this._maxHealth = health;
+
+  this.getPack = function() {
+    return this._pack;
+  };
+  this.getMaxHealth = function() {
+    return this._maxHealth;
+  };
```

Comment:
Update the references to `_pack` and `_maxHealth` so that they look at the private variables rather than the public ones on `this`.